www/templates/build.html

```diff
@@ -359,8 +359,22 @@ <h2 class="head-2">
             <div id="artifacts-js">
                 No artifacts
             </div>
-
-
+            <div class="divider"></div>
+            <div>
+                <a class="small-head"
+                {% if request.args %}
```

Comment:
I would avoid this way of html constructions, because it leads to heavy syntax and difficulties in catch errors. If possible I would make the 'href' variable and put there if-else statement or use some functions that cleverly joins url